document/core/appendix/algorithm.rst

```diff
@@ -122,9 +123,16 @@ The control stack is likewise manipulated through auxiliary functions:
      ctrls.pop()
      return frame
 
-   func label_types(frame : ctrl_frame) : list(val_types) =
+   func label_val_types(frame : ctrl_frame) : list(val_type) =
```

Comment:
Why rename the function?

Reply:
It was in the spirit of now calling [such labeltypes](https://webassembly.github.io/exception-handling/core/valid/conventions.html#syntax-labeltype) "label types" , as requested [in this comment](https://github.com/WebAssembly/exception-handling/pull/222#issuecomment-1249079017) but I see now that I may have overdone it. :) Reverting/removing as suggested.

Reply:
done